lona/html/attribute_dict.py

```diff
@@ -166,8 +168,10 @@ def to_attribute_string(self, skip_value=False):
                 if skip_value and key == 'value':
                     continue
 
-                if value == '':  # boolean properties
-                    string.append(key)
+                # boolean properties
+                if key in BOOLEAN_ATTRIBUTES:
```

Comment:
I think it's better to fix `CheckBox` instead of patching `AttributeDict`.

Reply:
My understanding is that these attribute names are boolean on every element type

Reply:
You a right